dist/jose-jwe.js

```diff
@@ -27,6 +27,7 @@
 JoseJWE = function() {
   this.setKeyEncryptionAlgorithm("RSA-OAEP");
   this.setContentEncryptionAlgorithm("A256GCM");
+  this.headers = {};
```

Comment:
Call this something like `userHeaders` to distinguish it from the headers managed by the library.